IssuesList.qml

```diff
@@ -43,11 +43,11 @@ ColumnLayout {
             property int rightMargin: 15
 
             visible: {
-                if (modelData.severity === "error" && showErrors.checked) {
+                if (modelData.severity === "error" && settings.showErrors.checked) {

```

Comment:
Maybe add showErrors, showWarnings and showInfo properties to IssuesList and bind them to `settings` in Main.qml? It's a bit more typing but provides better encapsulation.

Reply:
Done

Reply:
Done